metecho/api/tests/models.py

```diff
@@ -842,6 +859,18 @@ def test_notify_delete(self, scratch_org_factory):
 
             assert async_to_sync.called
 
+    def test_delete_scratch_org_with_no_owner(self, scratch_org_factory):
```

Comment:
I think we don't need this test, the case when `user=None` is already handled by the test on `User.delete()`